### ISSUE TITLE
💡 FEATURE REQUEST: Ok Google Request.

### ISSUE THREAD

**Issue by IceVDev** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#423]
**Is your feature request related to a problem? Please describe.**
    Well, Kinda I just want to have an automated thing like on the phone, when you say ok google it goes, so whenever I'm in another room or something I could still reach it.

**Describe the solution you'd like**
Could you maybe program a voice recognition thing like on the phone?

**Describe alternatives you've considered**
None, really :/

**Additional context**
No not really.

**Comment by IceVDev** [comment]
It is not really a bug but yeah.

**Comment by sumeshir26** [comment]
@IceVDev This has been implemented as of #373. If you want this feature now, you can [build this app yourself](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/blob/master/CONTRIBUTING.md#how-to-build).
Or, you can wait for the next update which will be released very soon.

**Comment by IceVDev** [comment]
@sumeshir26 I am not that advanced in this type of coding could someone help me out?

**Comment by sumeshir26** [comment]
@IceVDev To clone and run this repository you'll need [Git](https://git-scm.com/l) and [Node.js](https://nodejs.org/en/download/)  installed on your computer. 
If you are on Windows, you also need [Microsoft Visual Studio](https://visualstudio.microsoft.com/downloads/).
From PowerShell as **Administrator**:

```
# Install Windows Build Tools
npm install --global windows-build-tools
# Clone this repository
git clone https://github.com/electron/electron-quick-start
# Go into the repository
cd electron-quick-start
# Install dependencies
npm install
# Run the app
npm start
```

**Comment by Melvin-Abraham** [comment]
Implemented in #373
_Closing as resolved_
